modules/getting-started-cli-creating-new-project.adoc

```diff
@@ -1,34 +1,35 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-creating-new-project_{context}"]
 = Creating a new project
 
-A project enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes namespaces. Projects have additional features that enable user self-provisioning.
+A _project_ enables a community of users to organize and manage their content in isolation. Projects are {product-title} extensions to Kubernetes _namespaces_. Projects have additional features that enable user self-provisioning. Each project has its own set of objects, policies, constraints, and service accounts.
 
-Users must receive access to projects from administrators. Cluster administrators can allow developers to create their own projects. In most cases, users automatically have access to their own projects.
+Cluster administrators can allow developers to create their own projects. In most cases, users automatically have access to their own projects. Administrators can give users access to other projects as needed.
 
-Each project has its own set of objects, policies, constraints, and service accounts.
+This procedure creates a new project called `user-getting-started` that is used to complete this tutorial.
```

Comment:
```suggestion
Complete the following steps to create a project called `user-getting-started`. You will use the project throughout the rest of the tutorial.
```

Reply:
Per other comment, will leave the self-referential text. But I might take your suggestion to break it up into these two sentences!